### PR TITLE
Various dompin

### PR DESCRIPTION
**Pull request by ethteck**
Finishes hammer.c and matches some various things including some previous non-matchings

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 13 functions (+0.23%) on `us`.

✅ This PR fixes 3 warnings!

File: src/167570.c [hunk 1]
```diff
@@ -115,9 +118,226 @@ void filemenu_yesno_init(MenuPanel* tab) {
...
+                    case 0:
+                        filemenu_8024C098 = 2;
+                        filemenu_menus[filemenu_8024C098]->page = 0;
+                        gWindows[47].width = 182;
+                        gWindows[47].height = 25;
+                        gWindows[47].pos.x = (gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].width / 2) - 91 :
+                                             69;
+                        gWindows[47].pos.y = ((gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].height / 2) : 120) - gWindows[47].height / 2;
```
Comment (nanaian) [inline src/167570.c:170]: These ternaries are hard to read imo, I'd write them as something like
```suggestion
                        gWindows[47].pos.x = gWindows[47].parent != -1
                            ? gWindows[gWindows[47].parent].width / 2) - 91
                            : 69;
                        gWindows[47].pos.y = (
                            gWindows[47].parent != -1
                                ? (gWindows[gWindows[47].parent].height / 2)
                                : 120
                        ) - gWindows[47].height / 2;
```

File: src/167570.c [hunk 2]
```diff
@@ -115,9 +118,226 @@ void filemenu_yesno_init(MenuPanel* tab) {
...
+                    case 3:
+                        filemenu_8024C098 = 2;
+                        filemenu_menus[filemenu_8024C098]->page = 2;
+                        gWindows[47].width = 154;
+                        gWindows[47].height = 39;
+                        gWindows[47].pos.x = (gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].width / 2) - 77 :
+                                             83;
+                        gWindows[47].pos.y = ((gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].height / 2) : 120) - gWindows[47].height / 2;
```
Comment (nanaian) [inline src/167570.c:193]: " ternaries

File: src/167570.c [hunk 3]
```diff
@@ -115,9 +118,226 @@ void filemenu_yesno_init(MenuPanel* tab) {
...
+                    case 1:
+                        filemenu_8024C098 = 2;
+                        filemenu_menus[filemenu_8024C098]->page = 1;
+                        gWindows[47].width = 153;
+                        gWindows[47].height = 25;
+                        gWindows[47].pos.x = (gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].width / 2) - 76 :
+                                             84;
+                        gWindows[47].pos.y = ((gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].height / 2) : 120) - gWindows[47].height / 2;
```
Comment (nanaian) [inline src/167570.c:210]: "

File: src/167570.c [hunk 4]
```diff
@@ -115,9 +118,226 @@ void filemenu_yesno_init(MenuPanel* tab) {
...
+                        set_window_update(WINDOW_ID_50, (s32)filemenu_update_hidden_name_confirm);
+                        filemenu_8024C098 = 2;
+                        filemenu_menus[2]->page = 3;
+                        gWindows[47].width = 184;
+                        gWindows[47].height = 25;
+                        gWindows[47].pos.x = (gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].width / 2) - 92 :
+                                             68;
+                        gWindows[47].pos.y = ((gWindows[47].parent != -1) ?
+                                             (gWindows[gWindows[47].parent].height / 2) : 120) - gWindows[47].height / 2;
```
Comment (nanaian) [inline src/167570.c:253]: asdfg how does this appear like 4 times lol

File: src/world/action/hammer.c [hunk 5]
```diff
@@ -156,8 +265,149 @@ void action_hammer_update(void) {
...
+            yaw += 30.0f;
+        }
+    }
+
+    sin_cos_rad(yaw * TAU / 360.0f, &outSinTheta, &outCosTheta);
+    playerX = playerStatus->position.x;
+    playerY = playerStatus->position.y;
+    playerZ = playerStatus->position.z;
+
+    for (i = 1; i < 0x10; i++) {
```
Comment (nanaian) [inline src/world/action/hammer.c:308]:
```suggestion
    for (i = 1; i < 16; i++) {
```

File: src/world/action/hammer.c [hunk 6]
```diff
@@ -156,8 +265,149 @@ void action_hammer_update(void) {
...
+            angle = clamp_angle(yaw + 90.0f - gCameras[gCurrentCameraID].currentYaw);
+            if (angle >= 90.0f && angle < 270.0f) {
+                yaw += 15.0f;
+            } else {
+                yaw += -15.0f;
+            }
+        }
+        sin_cos_rad(yaw * TAU / 360.0f, &outSinTheta, &outCosTheta);
+
+        for (i = 1; i < 0x10; i++) {
```
Comment (nanaian) [inline src/world/action/hammer.c:339]:
```suggestion
        for (i = 1; i < 16; i++) {
```

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**